azurelinuxagent/common/cgroups/cgroups.py

```diff
@@ -474,6 +340,30 @@ def _try_mkdir(path):
                 else:
                     raise
 
+    @staticmethod
+    def _try_rmdir(path):
+        """
+        Try to create a directory, recursively. If it already exists as such, do nothing. Raise the appropriate
```

Comment:
comment copy and pasted from somewhere else?

Reply:
I'll fix the comment.